frame/bridge/s2s/issuing/src/lib.rs

```diff
@@ -102,7 +102,7 @@ pub mod pallet {
 		#[pallet::weight(
 			<T as Config>::WeightInfo::asset_burn_event_handle()
 		)]
-		#[frame_support::transactional]
+		#[transactional]
```

Comment:
This one is necessary, right? Since we perform a `transfer` before `burn_and_remote_unlock`.

Reply:
yes